file_paths.md

```diff
@@ -197,13 +210,25 @@ user or group.
             puppet.conf *                     # :config
             routes.yaml                       # :route_file
             ssl                               # :ssldir
-    -------------------------------------------------------------------------------------------------------               
+    -------------------------------------------------------------------------------------------------------
         var *
             log                               # :logdir                      same
                 puppet.log                    # not enabled by default
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.conf                    # pxp-agent configuration file
```

Comment:
Without the PCP configuration (server WebSocket URL, cert paths), pxp-agent can't start. The required parameters are specified in the [README](https://github.com/puppetlabs/pxp-agent). 

We discussed about removing the config file requirement introduced in [CTH-345](https://tickets.puppetlabs.com/browse/CTH-345) in favour of the previous behaviour (exit if the entire PCP configuration was not provided).